tests/BambuserTest.php

```diff
@@ -7,14 +7,14 @@ class BambuserTest extends AbstractTestCase
     public function testOne()
     {
         $this->assertEmbed(
-            'https://bambuser.com/v/7088053',
+            'https://bambuser.com',
```

Comment:
Seems like bambuser removed its video stream service so I'd remove this test and all related with bambuser.

Reply:
I will remove this releated test.